test/e2e/framework/convenience.go

```diff
@@ -0,0 +1,43 @@
+// +build e2e
```

Comment:
what do you mean by `convenience.go` here with the file name?

Reply:
I think it's mostly just helpers but I'll probably rename it to `scheme.go` as that makes the most sense.